tests/cucumber/features/step_definitions/organizations_steps.coffee

```diff
@@ -27,6 +33,7 @@ do ->
 
     @When /^I click on the organization link$/, (callback) ->
       @browser
+        .pause(1000)
```

Comment:
Increasing the timeout on waitForVisible seems to make it pass more often, but it's still flakier than pausing first (even if the timeout is longer than the pause).